scss/_buttons.scss

```diff
@@ -150,6 +150,39 @@
     @include button-outline-variant($value);
   }
 }
+
+@each $color, $value in $theme-colors {
+  .btn-subtle {
```

Comment:
So should I try making a new button ??

should the code be like?
`@each $color, $value in $theme-colors {`
`
  .btn-tert {`
`
    @include button-variant($value, $value);
 `
`  }`
`
}`


I am kinda beginner so please don't mind me

Reply:
TBH, even if I think that this feature is needed, this might be studied a bit more, I think that this kind of feature might be thought for whole Bootstrap at the same time at a naming and code level. 

Let's continue the discussion in the issue, and when everyone agrees on the solution, let's create a PR.